packages/nuxt-windicss/src/module.ts

```diff
@@ -11,39 +11,22 @@ import {
   importModule,
   requireModulePkg,
   requireModule,
-  tryRequireModule,
   isNuxt3,
   extendWebpackConfig,
 } from '@nuxt/kit-edge'
 import type { File } from '@nuxt/content/types/content'
+import { version } from '../package.json'
```

Comment:
Unbuild now treeshakes `package.json` imports! Just before publishing (and when version is bumped) with `prepack` script we build dist with release version.